platforms/webextension/content-communication-manager.es

```diff
@@ -36,7 +36,9 @@ function handleRequest(message, sender) {
     return undefined;
   }
 
-  return inject.module(module).action(action, ...args, sender);
+  inject.module(module).action(action, ...args, sender);
```

Comment:
How is the content script getting the response back in we do not return the promise?

Reply:
Good catch! For instance, adblocking fails then on `https://spiegel.de/`.

The warning does originate from this function, so it is the right place. But it is correct that returning the promise is important.

Reply:
Right, i missed, that returned promise value works like `sendResponse`.. but what then it means that some of the sendMessage origin does not wait for the response. I suppose, that they don't care about it, but the above code sens a promise always, which causes the issue

Reply:
Observation: in the debugger, I see that we are using the polyfill:

https://github.com/mozilla/webextension-polyfill/blob/780518ed1d9b05e6b31c4067d4db29927779abf3/src/browser-polyfill.js#L426

My guess is that we are using the API correctly, but the version of the polyfill creates issues on modern Chrome version. (?)